README.md

```diff
@@ -95,35 +95,65 @@ These operations add to or update data in Immutable's databases and typically re
 Signers are abstractions of user accounts that can be used to sign transactions. A user's private key is required to generate them.
 
 There are two ways to get signers in your application:
-
 1. [Generate your own by obtaining and using the user's private keys](#1-generate-your-own-signers)
 2. [Use our Wallet SDK to connect to a user's wallet application](#2-generate-signers-using-the-wallet-sdk)
 
 The first option, where an application obtains a user's private key directly, is risky because these keys allow anyone in possession of them full control of an account.
 
 The second option provides an application with an interface to the user's account by prompting the user to connect with their wallet application (ie. mobile or browser wallet). Once connected the app can begin asking the user to sign transactions and messages without having to reveal their private key.
 
-As Immutable X enables applications to execute signed transactions on both Ethereum (layer 1) and StarkEx (layer 2), signers are required for both these layers.
+### Pre-requisite: User must have Ethereum (L1) and Stark (L2) keys
 
-### 1. Generate your own signers
+As Immutable X enables applications to execute signed transactions on both Ethereum (layer 1) and StarkEx (layer 2), signers are required for both these layers. In order to generate an Ethereum or Stark signer, a user's Ethereum or Stark private key is required.
```

Comment:
As the ImmutableX Core SDK enables applications to execute signed transactions on Ethereum (Layer 1) and StarkEx (Layer 2), signers are required for both these layers.  To generate an Ethereum or Stark signer, a user's Ethereum or Stark private key is required.

Reply:
Would be good to list which operations where L1 Signer alone sufficient and where both L1 and L2 Signers are required.

Reply:
If we look at the endpoints it provides a list where L1 and L2 signatures are required.